### PR TITLE
Add EVM page

### PR DESCRIPTION
**Pull request by chasefleming**
Addresses: https://github.com/onflow/docs/issues/512

### CODE REVIEW

**Comment by Aliserag** [comment]
Something strange happening with the [URL](https://www.loom.com/share/45486745e6e34dce9c51deaae229ad3e?sid=21cb693f-37b1-43e8-bb7c-e5fe7a976a93)

File: docs/build/evm.md [hunk 1]
```diff
@@ -0,0 +1,45 @@
+---
+title: Flow EVM
+sidebar_label: Flow EVM
+sidebar_position: 10
+---
+
+# About EVM on Flow
+
+Flow is a secure Layer 1 blockchain committed to driving Web3 adoption into the mainstream. The team behind the blockchain not only coined the term NFT but made strides in opening up web3 to the world through Disney Pinnacle, NBA Topshot and Doodles. A significant milestone in this journey is the introduction of Ethereum Virtual Machine (EVM) equivalency i 2024.
```
Comment (gregsantos) [inline docs/build/evm.md:9]:
```suggestion
Flow is a secure Layer 1 blockchain committed to driving Web3 adoption into the mainstream. The team behind the blockchain not only coined the term NFT but made strides in opening up web3 to the world through Disney Pinnacle, NBA Topshot and Doodles. A significant milestone in this journey is the introduction of Ethereum Virtual Machine (EVM) equivalency in 2024. 
```

File: docs/build/evm.md [hunk 2]
```diff
@@ -0,0 +1,45 @@
...
+
+### **Seamless Integration for Ethereum Developers**
+
+With EVM equivalency, Solidity developers from the Ethereum ecosystem can effortlessly deploy their smart contracts on Flow with no changes to their code. We’re currently building up partnerships so that all one’s favorite developer tools, oracles, and platforms are compatible on day one.
+
+**Key Advantages**
+
+- **Access New Markets**: Easily tap into Flow's user base and unique IPs without any implementation risk.
+- **Simplified Multi-Chain Experience**: Ideal for applications without a multi-chain approach or those wanting to ensure cross-chain compatibility.
+- **Leverage Flow’s Protocol**: Expand EVM capabilities and transcend EVM limitations using Cadence, which offers a powerful new account model, programmable resources and hybrid ownership.
```
Comment (gregsantos) [inline docs/build/evm.md:19]:
```suggestion
- **Leverage Flow’s Protocol**: Expand EVM capabilities and transcend EVM limitations using Cadence, which offers a powerful new account model, programmable resources, and hybrid ownership.
```

**Review by gregsantos — CHANGES_REQUESTED** [review]

File: docs/build/evm.md (hunk 1 above)
Comment (franklywatson) [reply docs/build/evm.md:9]: Is it really necessary to rehash the history here?

**Review by gregsantos — APPROVED** [review]

**Pull request merged**